python/tskit/trees.py

```diff
@@ -8593,138 +8593,188 @@ def genetic_relatedness_vector(
         return out
 
     def pca(
-            self,
```

Comment:
I rearranged these to better match other methods (e.g., `windows` always come first, so I had it first after `n_components`)